test/Microsoft.AspNetCore.Antiforgery.Test/Internal/DefaultAntiforgeryTest.cs

```diff
@@ -276,6 +277,67 @@ public void GetAndStoreTokens_ExistingValidCookieToken_NotOverriden()
         }
 
         [Fact]
+        public void GetAndStoreTokens_ExistingValidCookieToken_NotOverriden_AndSetsDoNotCacheHeaders()
```

Comment:
Is this really the right behaviour? Should this be different from `SetCookieTokenAndHeader`?